addons/mail/static/tests/chat_window/chat_window_tests.js

```diff
@@ -1001,3 +1001,20 @@ QUnit.test("Chat window in mobile are not foldable", async (assert) => {
     await nextAnimationFrame();
     assert.containsOnce($, ".o-mail-ChatWindow-content"); // content => non-folded
 });
+
+QUnit.test("Open chat window of new inviter", async (assert) => {
+    const pyEnv = await startServer();
+    await start();
+    const partnerId = pyEnv["res.partner"].create({ name: "Newbie" });
+    pyEnv["res.users"].create({ partner_id: partnerId });
+    // simulate receiving notification of new connection of inviting user
+    pyEnv["bus.bus"]._sendone(pyEnv.currentPartner, "res.users/connection", {
+        username: "Newbie",
+        partner_id: partnerId,
```

Comment:
python sends `'partnerId': user.partner_id.id,` (camel case)

Reply:
note that it would be nice if the python also sent the user id with the partner data so that we don't need to make the extra search rpc to find existing user matching partner

Reply:
Changes done (`partnerId` instead of `partner_id`).

Indeed would be nice to have user info when receiving partner data.
In general, there's no guarantee we receive this partner knowledge with user (could come from `name_get`, which gives just id and name of partner).
So that's why it's still safer to keep fetching user if we lack this info.